src/app/loadout-builder/utils.ts

```diff
@@ -111,10 +402,10 @@ export function getBaseStatValues(
     for (const socket of item.sockets.sockets) {
       const plugHash = socket?.plug?.plugItem?.hash ?? NaN;
 
-      if (socket.plug?.stats && !masterworkSocketHashes.includes(plugHash)) {
+      if (socket.plug?.stats && masterworkSocketHashes.includes(plugHash)) {
         for (const statHash of statValues) {
           if (socket.plug.stats[statHash]) {
-            baseStats[statHash] -= socket.plug.stats[statHash];
+            baseStats[statHash] += socket.plug.stats[statHash];
```

Comment:
I was gonna blindly approve this PR for being just moves/renames - but what is this stuff all about? Are there more changes like this?

Reply:
No this is the only one. This is the part I was referring to in the last sentence on the description. This keeps it in sync with the same function in process. I am using base stats now so this keeps them in line

Reply:
On second thought I will pull that change out and do a larger cleanup for those methods. At the moment they have code in there for both armour 1.0 and 2.0. This function only gets run for the perk combinations on 1.0 so I think I can clean it up so it is more maintainable.